cmsl1t/collections/efficiency.py

```diff
@@ -1,8 +1,8 @@
 """
-.. module:: collections.efficiency
-    :synopsis: Module for creating efficiency(turnon)-curves
+    .. module:: collections.efficiency
+        :synopsis: Module for creating efficiency(turnon)-curves
 
-.. moduleauthor:: Luke Kreczko
+    .. moduleauthor:: Luke Kreczko
```

Comment:
Do we plan to include such an author line in all files?  I personally find this sort of thing clumsy: often it gets left behind when files are updated, and it's not clear who the author is when several people contribute, or one works for a long time, then leaves and someone else takes over.  Since the information (and more) can be taken from git directly, I think it's better without these sorts of lines and just have people use `git annotate` if they want to see who's been working on a file most recently.